content/en/api/synthetics/post_test.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Create a test
+type: apicontent
+order: 24.1
+external_redirect: /api/#create-tests
+---
+
+## Create a test
+
+Create a Synthetics test to initiate and configure the tests you want Datadog to send to your API endpoints or to your browser app. You can configure the endpoints being tested, the number of tests, and where they are coming from.
+
+##### Arguments
+
+*   **`assertions`** - _required_ - The function where you define when you want Datadog to alert you. Assertions are defined by: `operator`, `property`, `type`, and `target`.
```

Comment:
I found the definition a bit unclear. How about `assertions`: a list of assertions on the response of the test that should pass for the test to be considered successful. Each assertion has a `type`, an `operator`, a `target`, and possibly a `property`.